content/html/concepts/attributes/attributes.md

```diff
@@ -13,9 +13,9 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
```

Comment:
Fix the spacing above for Tags and 'learn-html'

Reply:
This appears to be fixed now, but I'm confused as to why it appears this way in the code review. If there's something I haven't done right, please let me know. Otherwise I think everything is as it should be. Thanks.